app/allocation/views/view.njk

```diff
@@ -0,0 +1,97 @@
+{% extends "layouts/two-column.njk" %}
+
+{% set additionalContainerClass = "sticky-sidebar-container" %}
+
+{% block pageTitle %}
+  {{ t("allocations::view.page_title") }}
+{% endblock %}
+
+{% block beforeContent %}
+  {{ govukBackLink({
+    text: t("actions::back_to_dashboard"),
+    classes: "app-print--hide",
+    href: dashboardUrl
+  }) }}
+
+  <a href="javascript:window.print()" class="app-!-float-right app-print--hide govuk-!-margin-top-3 app-icon app-icon--print">
+    {{ t("actions::print_move_detail") }}
+  </a>
+
+  {{ super() }}
+{% endblock %}
+
+{% block contentHeader %}
+  <header class="govuk-!-margin-bottom-8 govuk-!-padding-left-0 govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l govuk-!-margin-bottom-1">
+      {{ t("allocations::view.heading", {
+        person_or_people: "person" | pluralize(allocation.resultCount),
+        moves_count: allocation.moves_count,
+        from_location: allocation.from_location.title,
+        to_location: allocation.to_location.title,
+        date: allocation.date | formatDate
+      }) | safe }}
```

Comment:
Does this need to be unescaped using `safe`? Doesn't look like there's any HTML in the translation

Reply:
Following up on this - do we need to use `| safe`?